test/e2e/specs/site-editor/block-style-variations.spec.js

```diff
@@ -149,12 +149,14 @@ test.describe( 'Block Style Variations', () => {
 				},
 			},
 		} );
-		// The save button has been re-enabled.
+
+		// Wait for the save button to be re-enabled.
```

Comment:
No blockers here, just making sure I understand the change here:

This change is now checking that that `Save` is visible, rather than enabled. From looking at the editor manually, while saving, the button says "Saving" rather than "Save", so `.toBeVisible()` is a good check to use, as it'll only pass if the button says "Save" rather than "Saving". Also, once the revision has been saved the button is in a disabled state, so `toBeVisible` is a more reliable check than `toBeEnabled`.

Assuming that's all how it's meant to be, that sounds great to me!

Reply:
Exactly what you said, thanks for confirming. 

Much better expressed than what my caveman brain was thinking: "Wait for save. Save there. Good save."